### PR TITLE
T1641

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1641-cornfield-test-failing-after-login

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2ad985ae5fe7c2d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2ad985ae5fe7c2d/output.txt

Total script time: 0.46 mins

Server is running on http://pr586.butyr.org/templates/test.html

File: cornfield/app.js [hunk 1]
```diff
@@ -352,6 +352,7 @@ app.get('/api/project/:id?', function(req, res) {
         var projectJSON = JSON.parse( project.data );
         projectJSON.name = project.name;
         projectJSON.projectID = project._id;
+        projectJSON.error = "okay";
```
Comment (secretrobotron) [inline cornfield/app.js]: This doesn't look entirely necessary. Is it? I tried removing it and it was fine.

Comment (mjschranz) [inline cornfield/app.js]: Jon seemed to want that back in when I pointed it out to him that it wasn't there anymore. Up to you if you want it in or not.

Comment (secretrobotron) [inline cornfield/app.js]: Ok, let's remove it. We should make it more concrete if the error property should be set on something. This is more of a hack than anything and we don't use it anymore.

Comment (jbuck) [inline cornfield/app.js]: Yeah, we can take this out now since we don't have the load button anymore.

**Pull request merged**